docs/pages/admin-guides/access-controls/sso.mdx

```diff
@@ -298,19 +282,19 @@ create a `cluster_auth_preference` resource.
   version: v2
   ```
 
-  Create the resource:
+Create the resource:
 
   ```code
   # Log in to your cluster with tsh so you can run tctl commands.
   $ tsh login --proxy=<Var name="mytenant.teleport.sh" /> --user=myuser
   $ tctl create -f cap.yaml
   ```
 
-  </TabItem>
-</Tabs>
+(!docs/pages/includes/sso/idp-initiated.mdx!)
```

Comment:
```suggestion
```
We already include this partial in the TabItems related to SAML connectors, and it doesn't make as much sense here before we introduce the `saml`, `oidc`, and `github` resources.